modules/sb-psc-attachment/variables.tf

```diff
@@ -46,4 +46,30 @@ variable "apigee_organization" {
     condition     = can(regex("^(organizations/[a-zA-Z0-9-_]+)$", var.apigee_organization))
     error_message = "Invalid Apigee Organization ID. Please use the format \"organizations/[a-zA-Z0-9-_]+\"."
   }
-}
\ No newline at end of file
+}
+
+variable "target_servers" {
```

Comment:
I'm thinking this should actually be moved to [modules/apigee-x-core](https://github.com/apigee/terraform-modules/tree/main/modules/apigee-x-core) it's not  PSC specific, is it?

Reply:
no it is not, but here's my reasoning to add it here: if you put it in the x-core, would you not have a cross dependency where you need to x-apigee module to create the target server, but then you need (in this case the psc) to get the ip to configure it? it's a catch 22.
this way, if i want to add one or more target server in one or more environments, the outptut is chained straight to the resource.

Reply:
Sorry, my bad I got excited about the target_server resource and overlooked the host binding to the endpoint. Makes perfect sense. We can consider adding the target servers to apigee-x-core in another PR.